### PR TITLE
https://issues.redhat.com/browse/ACM-12037

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: governance/policy_operator.adoc [hunk 1]
```diff
@@ -61,11 +61,13 @@ a| Define the configurations to create an operator subscription. Add information
 - `namespace`: If not specified, the default namespaced that is used for {ocp-short} managed clusters is `openshift-operators`.
 - `source`: If not specified, the catalog that contains the operator is chosen.
 - `sourceNamespace`: If not specified, the namespace of the catalog that contains the operator is chosen.
++
+*Note:* If you define a value for `installPlanApproval`, your policy becomes invalid.
+//does invalid==non-compliant?
```
Comment (JustinKuli) [inline governance/policy_operator.adoc]: Yes, invalid is a kind of noncompliant status. When an OperatorPolicy is invalid, its compliance event will describe the issue, and the overall policy will be NonCompliant.

**Review by JustinKuli — APPROVED** [review]
Looks good overall to me

File: governance/policy_operator.adoc (hunk 1 above)
Comment (dockerymick) [inline governance/policy_operator.adoc]:
```suggestion
*Note:* If you define a value for `installPlanApproval`, your policy becomes non-compliant.
```

Comment (dockerymick) [reply governance/policy_operator.adoc]: Thanks for confirming @JustinKuli I will update the statement to say "non-compliant" to be consistent with the verbiage

**Review by jc-berger — APPROVED** [review]
/lgtm

File: governance/policy_operator.adoc [hunk 2]
```diff
@@ -61,11 +61,12 @@ a| Define the configurations to create an operator subscription. Add information
...
 - `sourceNamespace`: If not specified, the namespace of the catalog that contains the operator is chosen.
++
+*Note:* If you define a value for `installPlanApproval`, your policy becomes non-compliant.
 
-
-| `subscriptions.installPlanApproval`
+| `spec.upgradeApproval`
 | Required
-| If the `installPlanApproval` field is set to `Manual`, and the `spec.versions` field is empty, you must manually patch the `.spec.approved` field in the relevant `InstallPlan` resource to proceed with the installation process. The `InstallPlan` resource for an operator is generated after the subscription is created by the controller, and represents the intention to install a specific version of the desired operator.
+| If the `upgradeApproval` field is set to `Automatic`, version upgrades on the cluster are approved by the policy when policy is set to `enforce`. If you set the field to `None`, version upgrades to the specific operator are not approved when the policy is set to `enforce`.
```
Comment (dockerymick) [inline governance/policy_operator.adoc]:
```suggestion
| If the `upgradeApproval` field is set to `Automatic`, version upgrades on the cluster are approved by the policy when the policy is set to `enforce`. If you set the field to `None`, version upgrades to the specific operator are not approved when the policy is set to `enforce`.
```

**Pull request merged**